nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -262,7 +264,11 @@ def forward_for_export(self, audio_signal, length=None):
         if self.att_context_size[1] >= 0:
             att_mask = att_mask.tril(diagonal=self.att_context_size[1])
         att_mask = ~att_mask
-        pad_mask = ~pad_mask
+
+        if self.use_pad_mask:
+            pad_mask = ~pad_mask
+        else:
+            pad_mask = None
```

Comment:
Just a note, None mode isn't used anywhere, is that ok ?

Reply:
What is the purpose of this use_pad_mask? It looks to be always True, right?

Reply:
I have prepared 'mask disabled' mode - I think you have suggested on Slack it could be skipped for inference - but numeric results were rather different so I left it disabled for now.